docs/platform/api/getting-started-with-the-linode-api-v4.md

```diff
@@ -0,0 +1,358 @@
+---
+author:
+  name: Jared Kobos
+  email: docs@linode.com
+description: 'This guide introduces v4 of the Linode API and demonstrates several basic queries. It also covers authentication and the process of creating a new Linode through the API.'
+keywords: ["linode api", "api v4", "access token", "api key"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2018-03-26
+modified_by:
+  name: Linode
+published: 2018-03-23
+title: Getting Started with the Linode API v4
+external_resources:
+  - '[API Documentation](https://developers.linode.com/v4/introduction)'
+---
+
+## What is the Linode API?
+
+Most Linode users interact with their accounts through either the [Linode Manager](https://manager.linode.com/) or the [CLI](https://github.com/linode/linode-cli). However, Linode also makes its back-end available in the form of a [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) API, so that almost any task that can be performed through the Manager or CLI can also be performed and automated through a series of HTTPS requests. This makes it possible to build applications that create Linodes, deploy images, check the status and usage of a user's Linodes, and perform other account maintenance.
+
+Version 4 of the Linode API is now available. This new version includes several powerful features such as advanced filtering, response pagination, and OAuth support.
+
+Currently, Linode maintains a [Python library](https://github.com/linode/python-linode-api) that serves as a wrapper around the new API. Unofficial bindings are available in other languages, and it is always possible to use the API to build a new library in the language of your choice.
+
+This guide demonstrates the major features of the API and provides an example of how to obtain an access token and create a Linode.
+
+## Top Level Endpoints - Quick Reference
+
+All calls to the Linode API begin with `https://api.linode.com/v4/`. In keeping with RESTful architecture, different areas of functionality like images, support tickets, networking, etc. are located at different endpoints.
+
+This table contains a list of the top level endpoints. Specific information about these endpoints is available in the [API documentation](https://developers.linode.com/v4/introduction):
+
+| Endpoint  | Description  |
+|---|---|
+| **/linodes**        | Create, edit, and retrieve information about Linodes, including Linode instances on your account, kernels, and Stackscripts.  |
```

Comment:
probably wouldn't advertise kernels here, maybe "Retrieve information about your Linodes, their backups and stats, and stackscripts?"